EIPS/eip-2020.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 2020
```

Comment:
Why is this numbered 2020?

Reply:
Just a suggestion for an easy to remember number. From my understanding the number doesn't necessarily have to correspond to the PR #.

Reply:
I also don't mind calling this EIP 1193 if we don't want to jump numbers.